tests/integration/test_crt.py

```diff
@@ -44,20 +51,59 @@ def on_done(self, **kwargs):
 class TestCRTS3Transfers(BaseTransferManagerIntegTest):
     """Tests for the high level s3transfer based on CRT implementation."""
 
+    def setUp(self):
+        super().setUp()
+        self.s3_key = 's3key.txt'
+        self.download_path = os.path.join(self.files.rootdir, 'download.txt')
+
     def _create_s3_transfer(self):
         self.request_serializer = s3transfer.crt.BotocoreCRTRequestSerializer(
-            self.session
+            self.session, client_kwargs={'region_name': self.region}
         )
         credetial_resolver = self.session.get_component('credential_provider')
         self.s3_crt_client = s3transfer.crt.create_s3_crt_client(
-            self.session.get_config_variable("region"), credetial_resolver
```

Comment:
It looks like this will pull in a fix that we made in v2 where we were using the session's region. However, we never directly set the region on the session even though we set the region to be `us-west-2` for the base test class.